evaluation_script/utils.py

```diff
@@ -35,56 +40,47 @@ def calculate_precision(self):
         """Return the precision given the number of true positives (tp) and false positives (fp).
         Precision is 1 when fp is zero as there are no spurious results, even if tp is zero.
         """
-        return float(self.tp) / (self.tp + self.fp) if self.fp != 0 else 1
+        self.precision = float(self.tp) / (self.tp + self.fp) if self.fp != 0 else 1
```

Comment:
The `precision`, `recall`, `fpr`, `accuracy`, and `f1` member variables are no longer needed. Just rename the `calculate_xyz` methods to `xyz` and have them return the calculation. Don't bother setting member variables.

Reply:
That is what I do in PR #13. I didnt know which one you'd prefer. Please review PR#13 and I will close this one.